config/ecs/config.go

```diff
@@ -0,0 +1,109 @@
+package ecs
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-tf-aws/config/common"
+)
+
+func init() {
+
+	config.Store.SetForResource("aws_ecs_cluster", config.Resource{
+		ExternalName: config.ExternalName{
+			// Note(turkenh): Seems like we have a case that breaks our
```

Comment:
@turkenh Could you open an issue about this? We might have to go back and implement `terraform import`

Reply:
https://github.com/crossplane-contrib/terrajet/issues/104